insteon_mqtt/device/Dimmer.py

```diff
@@ -84,33 +75,16 @@ def __init__(self, protocol, modem, address, name=None):
         self.group_map.update({0x01: self.handle_on_off})
 
     #-----------------------------------------------------------------------
-    def on(self, group=0x01, level=None, mode=on_off.Mode.NORMAL, reason="",
-           transition=None, on_done=None):
-        """Turn the device on.
-
-        NOTE: This does NOT simulate a button press on the device - it just
-        changes the state of the device.  It will not trigger any responders
-        that are linked to this device.  To simulate a button press, call the
-        scene() method.
-
-        This will send the command to the device to update it's state.  When
-        we get an ACK of the result, we'll change our internal state and emit
-        the state changed signals.
+    def cmd_on_values(self, mode, level, transition, group):
+        """Calculate Cmd Values for On
 
         Args:
-          group (int): The group to send the command to.  For this device,
```

Comment:
Fixed

Reply:
FWIW, I don't see any changes to this file in your recent commits.  It's also fine with me if the group argument goes undocumented here, since it's ignored, and there's only one valid group # for normal dimmers.